bier/serialization/TypeNode.py

```diff
@@ -26,6 +26,36 @@ def write_to(self, value, writer, context=None):
         return 0
 
 
+@dataclass(frozen=True)
+class MemberLengthNode(TypeNode[int]):
+    """
+    Returns the value of the member with the given name.
```

Comment:
I guess that it's fine for now,
long term we will have to change this though.

Long term being, when we introduce the option to either only parse specific fields, or alternatively skip specific fields. In either case, a class field could be referred which isn't within the context.